src/tracker/mime-type-tracker-query.vala

```diff
@@ -3,12 +3,16 @@
 public class Games.MimeTypeTrackerQuery : Object, TrackerQuery {
 	public delegate Game GameForUri (string uri) throws Error;
 
+	private const uint HANDLED_GAMES_PER_CYCLE = 5;
+
 	private string mime_type;
 	private GameForUri game_for_uri;
+	private string[] uris_for_source;
```

Comment:
Just "uris" or just "result", "game_for_uri" is named that way because it is a delegate actually performing a "conversion", this just store the uris which are the results of the query. So maybe "uris" will be better.